### ISSUE TITLE
Implement fetch.txt feature

### ISSUE THREAD

**Issue by nuest** [ropensci/datapack#59]
The BagIt spec defines an optional file `fetch.txt` which includes external references to (potentially large) files that are not actually in the bag, but must be downloaded before validation (#58).

https://tools.ietf.org/html/draft-kunze-bagit-13#section-2.2.3

The contents to be stored to `fetch.txt` could be passed in a simple datastructure to `serializeToBagit`.

Just an idea, looking forward to feedback.

**Comment by gothub** [comment]
@nuest So when a DataPackage is serialized, the only action needed is to include the optional fetch.txt. When a package is validated, then then contents of
fetch.txt are retrieved and checked against the manifest. Is this correct?

If this is the case, then this sounds like a great addition.

**Comment by nuest** [comment]
Yes, that's correct. I'll see if I can get to this some time later this year!
